src/pages/index.js

```diff
@@ -67,17 +126,52 @@ const cardPreview = new PopupWithImage({
 const newCardPopup = new PopupWithForm({
   popupSelector: "#add-modal",
   handleFormSubmit: (inputValues) => {
-    renderCard(inputValues);
```

Comment:
* According to the task you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.